docs/stylesheets/base/utilities.css

```diff
@@ -12,3 +12,13 @@
   gap: 20px;
   padding: 10rem;
 }
+
+.showcase-badge {
+  background-color: var(--secondary-400);
+  padding: 2.6rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}

```

Comment:
The code looks good, the preview is amazing! Only thing, the text on a dark background is quite hard to read. Maybe use brighter color for text here? A small observation, for the Action button-you have written toggle description.

Reply:
Thanks for pointing that out. Will do the required changes.